doc/interfaces/tracking_flow.rst

```diff
@@ -0,0 +1,180 @@
+file (e.g.: ``det_tracking_output``)::
+
+    mkdir det_tracking_output
+
+Then, to perform the deterministic tracking we will run the ``dipy_track``
+command as::
+
+    dipy_track recons-csa/peaks.pam5 stop_file/mask.nii.gz wm_mask/brain_mask.nii.gz --seed_density 2 --tracking_method "det" --out_dir "det_tracking_output"
+
+ClosestPeaks Tracking
+*********************
+
+We will first create the directory in which to save the output tractogram
+file (e.g.: ``det_tracking_output``)::
+
+    mkdir det_tracking_output
+
+Then, to perform the deterministic tracking we will run the ``dipy_track``
+command as::
+
+    dipy_track recons-csa/peaks.pam5 stop_file/mask.nii.gz wm_mask/brain_mask.nii.gz --seed_density 2 --tracking_method "cp" --out_dir "cp_tracking_output"
+
+------------------------------------
+Particle Filtering Tracking PAM Flow
+------------------------------------
+
+Particle Filtering Tracking (PFT) uses tissue partial volume estimation (PVE)
+to reconstruct trajectories connecting the gray matter, and not incorrectly
+stopping in the white matter or in the corticospinal fluid. It relies on a
+stopping criterion that identifies the tissue where the streamline stopped.
+If the streamline correctly stopped in the gray matter, the trajectory is kept.
+If the streamline incorrectly stopped in the white matter or in the
+corticospinal fluid, PFT uses anatomical information to find an alternative
+streamline segment to extend the trajectory. When this segment is found, the
+tractography continues until the streamline correctly stops in the gray matter.
+
+PFT finds an alternative streamline segment whenever the stopping criterion
+returns a position classified as 'INVALIDPOINT'.
+
+We will use the ``stanford_hardi`` dataset in DIPY to showcase this tracking
+method. As with any other workflow in DIPY, you can also use your own data!
+
+We will first create a directory in which to save the output tractogram file
+(e.g.: ``pf_track_output``) and seeding file (e.g.: ``wm_mask``) ::
+    
+    mkdir pf_track_output wm_mask
+
+To run the Particle Filtering Tracking method, we need to specify the paths
+to the diffusion input file, white matter partial volume estimate, grey matter
+partial volume estimate, and cerebrospinal fluid partial volume estimate for
+tracking, and seeding file followed by optional arguments. In this case, we
+will be specifying the threshold for the Probability Mass Function that controls
+the randomness or probabilistic nature of the tracking (``pmf_threshold``),
+and the output directory (``out_dir``).
+
+White matter, grey matter, and cerebrospinal fluid volume files are available
+through the ``stanford_pve_maps`` dataset.
+
+To get the seeding file, we will create the WM mask using median Otsu
+thresholding method by calling the ``dipy_median_otsu`` command on
+``stanford_pve_maps`` dataset::
+
+    dipy_median_otsu data/stanford_hardi/HARDI150.nii.gz pve_wm.nii.gz --out_dir "wm_mask"
```

Comment:
Same comment as above on the use of two input arguments and the PVE (both the explanation and the command).

Reply:
Addressed in d929e25